src/pybind/mgr/dashboard/frontend/tslint.json

```diff
@@ -45,7 +45,7 @@
     "object-literal-sort-keys": false,
     "one-line": [true, "check-open-brace", "check-catch", "check-else", "check-whitespace"],
     "prefer-const": true,
-    "quotemark": [true, "single", "avoid-escape"],
+    "quotemark": [true, "single"],
```

Comment:
It would be really great if `npm fix` would automatically correct the strings if wrong quotes  are in use. That said we should still include `avoid-escape`. But I'm not sure if tslint supports wrapping a string with backticks.

The rest looks good so far.

Reply:
`tslint` fixes the outside quotes automatically, but I don't think it can fix the ones inside.

I think removing `avoid-escape` reduces the situations where a bad combo could happen, so I think we should removed it.

 ![image](https://user-images.githubusercontent.com/399326/84701089-d89c1a00-af43-11ea-8127-901a0ece0cce.png)
![image](https://user-images.githubusercontent.com/399326/84701095-da65dd80-af43-11ea-8b5b-9b6850e757e5.png)

Reply:
After looking at the tracker issue to find out what bad situation you mend, I fully understand what you mean. I would have fixed it otherwise than, I would just replace double quotes with single quotes or the other way around inside the notification service, just to avoid code styling discussions in the future ;)

This approach is fine too, but it could lead to quoting styling discussions in PRs that do not mainly touch the topic of single or double quotes.